tap_quickbooks/__init__.py

```diff
@@ -34,6 +33,7 @@ def main():
         else:
             catalog = do_discover()
 
+        client = QuickbooksClient(args.config_path, config, args.dev)
```

Comment:
I would argue we shouldn't move this if we're also commenting out the actual test request in client.py. That will ensure that if we _do_ start seeing a wave of auth issues or poorly configured connections, we can go back to actually testing credentials during discovery and thus failing on connection check instead of allowing full configuration of a connection with bad creds